src/Math-AutomaticDifferenciation/PMDualNumber.class.st

```diff
@@ -1,26 +1,30 @@
 "
+In linear algebra, dual numbers extend the real numbers by adjoining one new element ε with the property ε^2 = 0 (ε is nilpotent).
+See here: https://en.wikipedia.org/wiki/Dual_number^
+
 PMDualNumbers can be used to calculate the first derivative if one  creates them this way:
 	PMDualNumber value: aNumber eps:  derivativeOfaNumber (1 if the derivative with respect to aNumber is calculated, 0 otherwise)
 
 PMDualNumbers can be mixed with Numbers.
 "
 Class {
 	#name : #PMDualNumber,
-	#superclass : #Magnitude,
+	#superclass : #Number,
 	#instVars : [
 		'value',
 		'eps'
 	],
 	#classVars : [
 		'ZeroApproximation'
 	],
-	#category : 'Math-AutomaticDifferenciation'
+	#category : #'Math-AutomaticDifferenciation'
 }
 
 { #category : #'instance creation' }
-PMDualNumber class >> value:aValue [
-"for entering a constant"
-^(self new value: aValue) eps: 0 
+PMDualNumber class >> value: aValue [
```

Comment:
If it's easy to change, perhaps we can rename this message to `constant:`. This would eliminate the need for the comment underneath.

Reply:
I will remove the constant comment. In fact, here, we just build a dual number where the coefficient of epsilon is 0. Maybe eps: can be rename as epsilon: